configure.py

```diff
@@ -1143,6 +1144,7 @@ def find_headers(repodir, excluded_dirs):
     #FIXME: we don't need all of scylla_core here, only the types module, need to modularize scylla_core.
     'tools/scylla-types': idls + ['tools/scylla-types.cc'] + scylla_core,
     'tools/scylla-sstable-index': idls + ['tools/scylla-sstable-index.cc'] + scylla_core,
+    'tools/scylla-sstable': idls + ['tools/scylla-sstable.cc', 'tools/schema_loader.cc'] + scylla_core,
```

Comment:
For the future: we won't be able to package those 80MB executables (with even larger debuginfo). So we'll need to multiplex everything into the main scylla binary.

The way to do it is to inspect argv[0] in main() (first thing, even before app_template), and if it's scylla-tool then dispatch to a multi-command tool_main that interprets the first non-option argv as sub-tool name (like git subcommands) and runs the requested command. If it's anything other than scylla-tool, run regular scylla.

Reply:
Yes, this is on my list, but first I want to tackle the tool mode issue, so that by the time we start shipping these they are actually usable on production systems.